source/includes/sharding-warning.rst

```diff
@@ -1,13 +1,13 @@
 
-.. warning::
-
-   Syncing large documents to a sharded destination cluster can cause 
-   the destination cluster to initiate chunk migration.  When the destination
-   cluster migrates chunks during sync, it may trigger a bug that can result
-   in data loss.
-
-   To avoid this, it is recommended that you run the :dbcommand:`balancerStop`
-   command on the destination cluster before starting a sync.  Once ``mongosync``
-   completes the sync, you can restart the balancer with the 
-   :dbcommand:`balancerStart` command.
-
+.. important::
+
+   If the destination is a sharded cluster with the balancer enabled, write 
+   performance of the destination cluster may be degraded during ``mongosync``'s 
+   Collection Copy phase. To warn of this potential performance impact,   
+   ``mongosync`` runs the :method:`sh.getBalancerState()` command during 
+   initialization. If it returns ``true``, it logs a warning. 
```

Comment:
[style guide] Would it be possible to [clarify the antecedents](https://www.mongodb.com/docs/meta/style-guide/writing/use-pronouns-carefully/) here? It may be unclear to readers what the first and second "it" in the last sentence refers to.